client/src/components/Mylist/MyAnimeList.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react'
+import { useAuth0 } from '@auth0/auth0-react'
 
 const MyAnimeList = () => {
-
+const {user} = useAuth0();
 /* make a get request for items in saved list
 fetch url ${user.sub}
 setLts(list)
 
 useEffect for authentication
 */
+const getSavedList = async () => {
+  //grab user info from auth0 id & sub
+  const userInfo = {
+    sub: user.sub,
+  }
+  console.log(userInfo);
 
+  const response = await fetch(`/myanimelist?sub=${user.sub}`)
```

Comment:
Awesome! This is a great step forward. Keep in mind this isn't any more secure than what I proposed of passing the user id in--in theory someone could call teh api with any sub, which is not a secret, but it makes sense to pass the subscriber id or email if you can (basically anything in auth0) and use that to look up the user in your db.

Reply:
Auth0 doesn't let me grab a user I'd (I was able to walk through this with another mentor) so I'm the backend we can call a user based on their sub which is unique in auth0! I sent a video on slack that shows what is happening here